octofree/docker-console.sh

```
@@ -0,0 +1,37 @@
+echo "║         OCTOFREE ENERGY MONITOR                    ║"
+echo "╚════════════════════════════════════════════════════╝"
+echo ""
+echo "Python app is running in the background..."
+echo ""
+echo "Options:"
+echo "  [1] Launch TUI Monitor (interactive dashboard)"
+echo "  [2] View logs (tail -f)"
+echo "  [3] Shell access"
+echo "  [q] Exit console"
+echo ""
+echo -n "Choose an option: "
```

Shellcheck (0.11.0)

[warning] In POSIX sh, echo flags are undefined.

(SC3037)